Projects/UOContent/Items/Weapons/Axes/BaseAxe.cs

```diff
@@ -12,17 +12,20 @@ public interface IAxe
         bool Axe(Mobile from, BaseAxe axe);
     }
 
-    public abstract class BaseAxe : BaseMeleeWeapon
+    [Serializable(2)]
```

Comment:
It's ok to reset to 0 here. Also needs a false since the version is not encoded in the Serialize.